README.md

```diff
@@ -0,0 +1,16 @@
+# Portfolio (STEP Project)
+
+## Introduction
+This is the repository hosting the files for the repository I built as my first project for the STEP program!
+Once deployed, I will attach a URL at which it can viewed.
+
+## Screenshots
+
+## Installation
+
+## Contributing
+Contributions and fixes are welcome. Before opening a PR with changes, please open an issue first to discuss anything you'd want to change!
+
+## License
```

Comment:
I believe the original repository contains a separate LICENSE file for this. No need to repeat here.

Reply:
Sure thing, I'll remove that section